pkg/rootless/rootless_linux.c

```diff
@@ -193,7 +193,11 @@ can_use_shortcut ()
 
   argv = get_cmd_line_args ();
   if (argv == NULL)
-    return false;
+    {
+      free (argv[0]);
```

Comment:
You are freeing an argv which is NULL, this will cause a crash.

Reply:
yes, commited wrong patch, sorry I will fix it right away